lib/trufi_app.dart

```diff
@@ -9,9 +9,11 @@ import './blocs/location_provider_bloc.dart';
 import './blocs/location_search_bloc.dart';
 import './blocs/preferences_bloc.dart';
 import './blocs/request_manager_bloc.dart';
+import './blocs/saved_locations_bloc.dart';
```

Comment:
Consider naming consistently: places (or locations) -- no one time locations and the other time places, isn't it the same?

Reply:
Yes, it is. I will make the modifications.